s3/pkg/datastore/azure/azure.go

```diff
@@ -67,7 +69,7 @@ func (ad *AzureAdapter) BucketCreate(ctx context.Context, input *pb.Bucket) erro
 	// Create the container on the service (with no metadata and no public access)
 	_, err := containerURL.Create(ctx, azblob.Metadata{}, azblob.PublicAccessNone)
 	if err != nil {
-		log.Error("failed to create bucket:",err)
+		log.Error("failed to create bucket:", err)
```

Comment:
One generic comment across services  : Most of the places we log the error/warning and Users don't have  a way of knowing what went wrong. UI is not getting / capturing any error scenarios well.. This could be a task for Next Rel. 
This should be handled in a generic way for all Multi-Cloud Services.

Reply:
Yes. This is one of the long pending enhancement. I will open issue to track.

Reply:
https://github.com/sodafoundation/multi-cloud/issues/1229